lxd/firewall/drivers/drivers_nftables.go

```diff
@@ -106,11 +106,11 @@ func (d Nftables) Compat() (bool, error) {
 
 // nftGenericItem represents some common fields amongst the different nftables types.
 type nftGenericItem struct {
-	Type   string // Type of item (table, chain or rule).
-	Family string `json:"family"` // Family of item (ip, ip6, bridge etc).
-	Table  string `json:"table"`  // Table the item belongs to (for chains and rules).
-	Chain  string `json:"chain"`  // Chain the item belongs to (for rules).
-	Name   string `json:"name"`   // Name of item (for tables and chains).
+	ItemType string `json:"-"`      // Type of item (table, chain or rule). Populated by LXD.
```

Comment:
Where do we actually populate it?

Reply:
Oh, I see, nevermind

Reply:
https://github.com/lxc/lxd/pull/7980/files/406d531d63b89ed48b5fcbd1067c8ab2d9619457#diff-738087ec5cfdc5a0312964e1f484c57fR143
https://github.com/lxc/lxd/pull/7980/files/406d531d63b89ed48b5fcbd1067c8ab2d9619457#diff-738087ec5cfdc5a0312964e1f484c57fR146
https://github.com/lxc/lxd/pull/7980/files/406d531d63b89ed48b5fcbd1067c8ab2d9619457#diff-738087ec5cfdc5a0312964e1f484c57fR149